src/mlpack/methods/ann/gan/gan.hpp

```diff
@@ -95,16 +96,25 @@ class GAN
   //! Move constructor.
   GAN(GAN&&);
 
+  /**
+   * Prepare the network for the given data.
+   * This function won't actually trigger training process.
+   *
+   * @param trainData The real data.
```

Comment:
Same here, the "real data", might be confusing.

Reply:
I have changed it to. `The data points of real distribution.`